src/Grpc.Net.Client/Internal/UserAgentGenerator.cs

```diff
@@ -44,7 +56,7 @@ internal static string GetUserAgentString()
             
             return GetUserAgentString(
                 processArch: RuntimeInformation.ProcessArchitecture,
-                runtimeVersion: IsMono ? null : Environment.Version, 
+                runtimeVersion: IsMono ? GetMonoRuntimeVersion(assembly) : Environment.Version, 
```

Comment:
this gets confusing. `Environment.Version` is actually the CLR Version (see docs https://docs.microsoft.com/en-us/dotnet/api/system.environment.version?view=net-5.0) so it might be less confusing to call the argument "clrVersion" rather than "runtimeVersion" since "runtimeVersion" gets easily confused with "runtimeInformation"

Reply:
Environment.Version has existed since .NET Framework 1.1 so it really should exist on mono.
What doesn't necessarily exist on mono is the RuntimeInformation.FrameworkDescription (only exists on .NET Framework 4.7.1, but that's hidden by the fact that we're currently targeting netstandard2.0 and not e.g. net461)